README.md

```diff
@@ -118,6 +118,7 @@ Compound cleaners don't clean data directly, but they _create_ cleaners that can
 - `asMap` - Builds a cleaner for an object used as key / value map.
 - `asOptional` - Builds a cleaner for an item that might be undefined or null.
 - `asEither` - Builds a cleaner for an item that might have multiple types.
+- `asMaybe` - Builds a cleaner for an item that may be a type.
```

Comment:
Maybe it "Builds a cleaner that quietly ignores invalid data."

Reply:
I concur